pkgs/tools/security/radamsa/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "radamsa-0.4";
+
+  src = fetchurl {
+    url = "http://haltp.org/download/radamsa-0.4.tar.gz";
+    sha256 = "1xs9dsrq6qrf104yi9x21scpr73crfikbi8q9njimiw5c1y6alrv";
+  };
+
+  patchPhase = ''
+    substitute ./tests/bd.sh ./tests/bd.sh \
+      --replace "/bin/echo" echo
+    substitute ./Makefile ./Makefile \
+      --replace "PREFIX=/usr" "PREFIX=$out" \
+      --replace "BINDIR=/bin" "BINDIR="
+  '';
+  
+  meta = {
+    description = "A general purpose fuzzer";
+    longDescription = ''Radamsa is a somewhat new kind of black-box fuzzer. It is designed to be a good general purpose tool for developers and vendors who want to test how well their products can withstand malicious inputs. Unlike most similar tools, it is intended to work for just about any kind of data without any extra hacking. It has been used to find previously unknown security vulnerabilities in handling of many kinds of file formats with very different structure, like bmp, png, gif, jpeg, svg, xml, ogg, avi, html, gz, bzip2, tiff, pdf and zip. The main goal is to make a tool that works well no matter what kind of data is thrown at it. By extension, we hope other fuzzers will not to find too many bugs in programs that developers have already fuzzed with radamsa.
```

Comment:
Please make lines in `longDescription` shorted, for example use line's width as 80 characters.